app/models/company.rb

```diff
@@ -0,0 +1,6 @@
+class Company < ActiveRecord::Base
+	belongs_to :subscription_package
+	has_one :company_setting
+	has_many :groups
+	has_many :users
```

Comment:
what about surveys, questions, responses and other tables?

Reply:
Surveys belong to users (who created them). Questions belong to surveys. Responses belong to questions.